src/transformers/models/auto/processing_auto.py

```diff
@@ -250,10 +252,24 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
         if type(config) in PROCESSOR_MAPPING:
             return PROCESSOR_MAPPING[type(config)].from_pretrained(pretrained_model_name_or_path, **kwargs)
 
+        # At this stage, there doesn't seem to be a `Processor` class available for this model, so let's try a
+        # tokenizer.
+        try:
+            return AutoTokenizer.from_pretrained(
```

Comment:
I guess the order doesn't matter too much here since if both tokenizer and feature extractor would be present then the model also should have a processor?

Reply:
Yes, I don't see when we could have both without a processor.